appium-uiautomator2-server/build.gradle

```diff
@@ -37,3 +37,17 @@ dependencies {
     androidTestCompile 'com.android.support.test.espresso:espresso-core:2.0'
 
 }
+
+task installAUT(type: Exec) {
```

Comment:
`connectedCheck` installing both the apks, but hangs down while executing tests under `androidTest` which contains an infinite [loop](https://github.com/appium/appium-uiautomator2-server/blob/master/appium-uiautomator2-server/src/androidTest/java/io/appium/uiautomator2/AppiumUiAutomator2Server.java#L27) for keeping the server up.